packages/react/src/components/header/components/headerUniversalBar/HeaderUniversalBar.tsx

```diff
@@ -48,7 +49,7 @@ export const HeaderUniversalBar = ({
   const childElements = getChildElementsEvenIfContainerInbetween(children);
 
   return (
```

Comment:
Is this a bug fix? Seems unrelated to linting errors.

Reply:
Ah yes this is a bug fix after all 👍🏻